portfolio/src/main/webapp/script.js

```diff
@@ -35,3 +35,36 @@ function createListElement(text) {
   liElement.innerText = text;
   return liElement;
 }
+
+/** Gets login status */
+function getLogin() {
+  getComments();
+  // hide comments
+
+  // fetch login status from servlet
+  fetch('/login').then(response => response.json()).then((json) => {
+    console.log('json from /login is ' + json);
```

Comment:
The comments are displayed even if I am not logged in. Is that the behavior you want?

If you want the comments to be shown only if the user is logged in, you should call getComments here. Ideally the server should also check that the user is logged in before returning comments.

Another issue I found is, that it shows me "login" even if I am logged in. You should probably check login status on "onload".

It  is ok to leave this as is and make changes if you want in a follow up PR.

Reply:
I fixed the comments visibility issue. So now previous comments are visible only when logged in.

About '"login" even if I am logged in', I checked again on the live server and was getting logout as the text. I am calling the getLogin() function as onload on the body tag. Is there anything else onload means?